app.js

```diff
@@ -1,9 +1,38 @@
 const express = require("express");
 const apiRouter = require("./routes/api-router");
-
+const dotenv = require("dotenv");
```

Comment:
Your `require("dotenv")` is still on line 3, as opposed to line 1. The `dotenv` library *must* be the first library included in the file. You can see this in the documentation here: https://www.npmjs.com/package/dotenv#usage